api/services.ts

```diff
@@ -2,7 +2,7 @@ import { AirtableClient, AirtableCreateResponse, AxiosClient } from './clients'
 import {
   Address,
   CreateServicesListRequest,
-  Service,
+  Resource,
```

Comment:
We should files and components with "Resource" too but we can do that in a separate PR since this one already has a lot of changes.